diff --git a/src/datatrove/tools/check_dataset.py b/src/datatrove/tools/check_dataset.py
--- a/src/datatrove/tools/check_dataset.py
+++ b/src/datatrove/tools/check_dataset.py
@@ -95,10 +95,13 @@ def check_dataset(
             assert last_token == eos_token or (chunk_size and read_count % chunk_size == 0), (
                 f"no EOS at doc end of doc {doci}"
             )
 
 
-if __name__ == "__main__":
+def main():
     args = parser.parse_args()
 
     input_folder: DataFolder = get_datafolder(args.data)
 
     check_dataset(input_folder, args.tokenizer, args.eos)
     print("All checks ok")
+
+if __name__ == "__main__":
+    main()
diff --git a/src/datatrove/tools/inspect_data.py b/src/datatrove/tools/inspect_data.py
--- a/src/datatrove/tools/inspect_data.py
+++ b/src/datatrove/tools/inspect_data.py
@@ -40,7 +40,7 @@
 )
 
 parser.add_argument(
-    "-s", "--sample", type=float, help="Randomly sample a given % of samples. 1.0 to see all samples", default=1.0
+    "-s", "--sample", type=float, help="Randomly sample a given %% of samples. 1.0 to see all samples", default=1.0
 )
 
 parser.add_argument(
